cmake/modules/FindHostTools.cmake

```diff
@@ -61,10 +65,6 @@ find_program(BOSSAC bossac)
 # in the mcuboot repository if that's present in some cases)
 find_program(IMGTOOL imgtool)
 
-find_package(Deprecated COMPONENTS XCC_USE_CLANG CROSS_COMPILE)
-
-find_package(Zephyr-sdk 0.15)
```

Comment:
See, moving this up was the right choice 🤣

Reply:
but you wanted the `0.15` to be moved up, not the whole line 😛